contracts/aggregator/avalanche/flashloan/helpers.sol

```diff
@@ -96,6 +117,71 @@ contract Helper is Variables {
         }
     }
 
+    /**
+     * @dev Supply token to aave.
+     * @notice Supply token to aave.
+     * @param _tokens token addresses.
+     * @param _amounts amounts of tokens.
+    */
+    function aaveSupply(address[] memory _tokens, uint256[] memory _amounts) internal {
+        uint256 length_ = _tokens.length;
+        require(_amounts.length == length_, "array-lengths-not-same");
+        for(uint256 i = 0; i < length_; i++) {
+            IERC20 token_ = IERC20(_tokens[i]);
+            token_.safeApprove(aaveLendingAddr, _amounts[i]);
+            aaveLending.deposit(_tokens[i], _amounts[i], address(this), 3228);
+            aaveLending.setUserUseReserveAsCollateral(_tokens[i], true);
```

Comment:
@Vaibhav20201 How does FLA support chain token or native token?

Reply:
It does not. It only supports erc20